src/Generating/FinishGen.cpp

```diff
@@ -14,6 +14,7 @@
 #include "../Simulator/FireSimulator.h"
 #include "../World.h"
 #include "../IniFile.h"
+#include "../BlockEntities/BlockEntity.h"
```

Comment:
Why has this include been added?

Reply:
I think it's a leftover from some experimentations (was trying to get the MaxCactusHeight variable from ini files). I'll remove it, as it's useless indeed :)